playbooks/robusta_playbooks/babysitter.py

```diff
@@ -79,3 +82,27 @@ def resource_babysitter(event: KubernetesAnyChangeEvent, config: BabysitterConfi
         title="Kubernetes Manifest Change",
     )
     event.add_finding(finding)
+
+
+class UrlParam(ActionParams):
+    """
+    :var url: url that should be used in the action
+    """
+
+    url: str
+    headers: Optional[Dict[str, str]] = None
+
+
+@action
+def json_change_tracker(event: KubernetesAnyChangeEvent, params: UrlParam):
+    """
+    post change json to the specified url
```

Comment:
I would change this since it shows up in the docs
```
Posts Kubernetes resource changes as JSON to a specified URL.
```

Reply:
good point, fixing